getting-started/vs-code-integration/new-project-wizard.md

```diff
@@ -36,25 +38,49 @@ The **Create New Project** wizard provides two options to start you project:
+The available templates vary depending on the target framework and hosting model. All of the above templates are available for the Client and Server projects for .NET 6 and .NET 7. The Hybrid projects and the .NET 8 WebApp support only the **Start from Blank App** option.
+
+### Section 2: Select Project Name and Path
+
+This section allows you to type your project name and select the output folder for it.
+
+### Section 3: Select UI for Blazor Version
+
+This section allows you to select the preferred UI for Blazor version. The current latest version is pre-selected by default. Therefore, you must ensure that your system has access to the same version of the UI components - you can achieve this with the [Telerik NuGet feed](../../installation/nuget).
+
+### Section 4: Select Hosting Model
+
+Here you can select the desired hosting model&mdash;[**WebApp**]({%slug getting-started/web-app%}), [**Client** (also: WebAssembly)]({%slug getting-started/client-side%}), [**Server**]({%slug getting-started/server-side%}), or [**Hybrid** (a hybrid Blazor MAUI)]({%slug getting-started/hybrid-blazor%}).
+
+The available target framework options are updated upon selecting the hosting model.
+
+### Section 5: Select Your License Type
 
-The wizard allows you to configure the following options:
+By default, the "Paid" option is selected corresponding to the commercial license type.
 
-* Name and output path for your app.
-* Hosting model type&mdash;[**WebApp**]({%slug getting-started/web-app%}), [**Client** (also: WebAssembly)]({%slug getting-started/client-side%}), [**Server**]({%slug getting-started/server-side%}), or [**Hybrid** (a hybrid Blazor MAUI)]({%slug getting-started/hybrid-blazor%}) app.
-* Your Telerik license (trial or commercial, depending on what you have). 
-* Whether to use localization in the project or not.
-* Whether to use font icons in the project or not.
-* The .NET version you want to use.
-* Sample pages that you can add to get started (available when `Start from Blank App` is selected).
-* The theme you want to apply - the main window provides a list of the [built-in (base) themes]({%slug general-information/themes%}) (Default, Bootstrap and Material). The `See All Swatches` button opens a separate menu with a list of [color swatches]({%slug themes-swatches%}) under the corresponding base theme.
+### Section 6: Choose How to Distribute the Client Assets
 
->caption Create New Project wizard
+This section allows you to choose whether to import the required Telerik UI stylesheet and JS Interop file by using the [static assets]({%slug getting-started/what-you-need%}#css-theme-and-javascript-files) or the [CDN method]({%slug common-features-cdn%}).
 
-![VS code wizard overview](images/vs-code-new-project-wizard.png)
+### Section 7: Enable or Disable Localization
 
->caption See All Swatches menu
+This section lets you specify whether to use [localization]({%slug globalization-localization%}) in the project or not.
```

Comment:
Add a note about the default selection and how enabling/disabling it impacts the projects (e.g., adds packages, increases size, and so on.)

```suggestion
Choose whether to use [localization]({%slug globalization-localization%}) in the project or not.
```